CustomerApp/src/main/java/com/brainscratch/laboratory/customerapp/Customer.java

```diff
@@ -0,0 +1,69 @@
+package com.brainscratch.laboratory.customerapp;
+
+public class Customer {
```

Comment:
Now Customer is model. As we discussed, models must be placed in the "model" package.

Reply:
done, i'll push updates now now